pkg/tfgen/generate_python.go

```diff
@@ -800,5 +800,59 @@ func pyName(name string) string {
 	}
 
 	components = append(components, string(currentComponent))
-	return strings.Join(components, "_")
+	result := strings.Join(components, "_")
+
+	// If the generated name clashes with a Python 2 or 3 keyword, add a trailing underscore per PEP 8:
+	// https://www.python.org/dev/peps/pep-0008/?#id46
```

Comment:
Nit: that URL doesn't lead directly to the recommendation. This URL does: https://www.python.org/dev/peps/pep-0008/?#function-and-method-arguments

Reply:
Ah, whoops, will fix.

Reply:
Ah, whoops, will fix.